### PR TITLE
feat: Upload Smoothie's metadata to Google PlayStore

### PR DESCRIPTION
**Pull request by teolemon**
### What
- [ ] Upload metadata to PlayStore
- [ ] Set up secrets
- [ ] Do the same for iOS

### Part of
- #5

### CODE REVIEW

File: .github/workflows/validate-metadata.yml [hunk 1]
```diff
@@ -2,16 +2,37 @@ name: Fastlane Metadata
 on:
   push:
     branches:
-      - main
+      - 'upload-to-playstore'
 jobs:
-  validate:
+    - name: Decrypt API JSON file
+      runs-on: ubuntu-latest
+      run: cd ./packages/smooth_app/android/fastlane/envfiles && chmod +x ./decrypt_secrets.sh && ./decrypt_secrets.sh
```
Comment (M123-dev) [inline .github/workflows/validate-metadata.yml]: Needs to be changed

File: .github/workflows/validate-metadata.yml [hunk 2]
```diff
@@ -2,16 +2,37 @@ name: Fastlane Metadata
 on:
   push:
     branches:
-      - main
+      - 'upload-to-playstore'
 jobs:
-  validate:
```
Comment (M123-dev) [inline .github/workflows/validate-metadata.yml:7]: We need a new job name

File: .github/workflows/validate-metadata.yml [hunk 3]
```diff
@@ -2,16 +2,34 @@ name: Fastlane Metadata
...
-    # required to run on Linux because this is a docker container action
-    name: validate
+  marketing:
     runs-on: ubuntu-latest
     steps:
+     - name: Decrypt API JSON file
+       run: cd ./packages/smooth_app/android/fastlane/envfiles && chmod +x ./decrypt_secrets.sh && ./decrypt_secrets.sh
+       env:
+        API_JSON_FILE_DECRYPTKEY: ${{ secrets.API_JSON_FILE_DECRYPTKEY }}
+        STORE_JKS_DECRYPTKEY: ${{ secrets.STORE_JKS_DECRYPTKEY }}
```
Comment (M123-dev) [inline .github/workflows/validate-metadata.yml:14]: Lets put the validation before the encryption

**Review by M123-dev — CHANGES_REQUESTED** [review]
Added a few comments

File: .github/workflows/validate-metadata.yml (hunk 3 above)
Comment (teolemon) [reply .github/workflows/validate-metadata.yml]: done

**Review by M123-dev — APPROVED** [review]
Besides the filename the the path to the credentials everything looks good by now 👍🏼

**Comment by teolemon** [comment]
Currently fails before the actual thing because of #6

File: .github/workflows/validate-metadata.yml [hunk 4]
```diff
@@ -2,16 +2,34 @@ name: Fastlane Metadata
 on:
```
[the remark this answers is not in the document]
Comment (teolemon) [reply .github/workflows/validate-metadata.yml]: done

File: .github/workflows/android-upload.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  marketing:
+    runs-on: ubuntu-latest
+    steps:
+    # required to run on Linux because this is a docker container action
+     - name: Checkout
+       uses: actions/checkout@v1
+     - name: Validate metadata
+       uses: ashutoshgngwr/validate-fastlane-supply-metadata@v1
+       with:
+        fastlaneDir: ./fastlane # optional. default is './fastlane'.
```
Comment (teolemon) [inline .github/workflows/android-upload.yml:16]:
```suggestion
        fastlaneDir: ./fastlane/metadata-android # optional. default is './fastlane'.
```

File: fastlane/Fastfile [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    end
+  end
+
+
+end
+
+
+
+
+
```
Comment (teolemon) [inline fastlane/Fastfile:38]:
```suggestion
```

File: .github/workflows/android-upload.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    steps:
+    # required to run on Linux because this is a docker container action
+     - name: Checkout
+       uses: actions/checkout@v1
+     - name: Validate metadata
+       uses: ashutoshgngwr/validate-fastlane-supply-metadata@v1
+       with:
+        fastlaneDir: ./fastlane/metadata-android # optional. default is './fastlane'.
+        # enable check to validate if a locale is supported by the Play Store Listing.
+        usePlayStoreLocales: true # optional. default is false.
```
Comment (teolemon) [inline .github/workflows/android-upload.yml:18]:
```suggestion
    # - name: Validate metadata
       #uses: ashutoshgngwr/validate-fastlane-supply-metadata@v1
       # with:
        # fastlaneDir: ./fastlane/metadata-android # optional. default is './fastlane'.
        # enable check to validate if a locale is supported by the Play Store Listing.
        # usePlayStoreLocales: true # optional. default is false.
```

File: fastlane/Appfile [hunk 8]
```diff
@@ -0,0 +1,4 @@
+# Android
+json_key_file(ENV['GOOGLE_PLAY_JSON_PATH'] || 'fastlane/envfiles/api-4712693179220384697-162836-33ea08672303.json')
+package_name(ENV['PACKAGE_NAME'] || "org.openfoodfacts.app")
+metadata_path("./fastlane/metadata-android")
```
Comment (teolemon) [inline fastlane/Appfile:4]:
```suggestion
```

File: fastlane/Fastfile [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+# update_fastlane
+
+
+default_platform(:android)
+
+platform :android do
+  desc "Deploy marketing"
+  lane :deploy_marketing do
+    begin
+      upload_to_play_store(
```
Comment (teolemon) [inline fastlane/Fastfile:23]:
```suggestion
      upload_to_play_store(
      metadata_path: "./fastlane/metadata-android"
```

File: fastlane/Fastfile [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+      metadata_path: "./fastlane/metadata-android"
+        skip_upload_metadata: false,
+        skip_upload_images: false,
```
Comment (teolemon) [inline fastlane/Fastfile:26]:
```suggestion
        metadata_path: "./fastlane/metadata-android",
        skip_upload_metadata: false,
        skip_upload_images: false,
```

**Pull request merged**